2022-01-20 16:27:32,661 INFO    MainThread:30500 [wandb_setup.py:_flush():71] setting env: {}
2022-01-20 16:27:32,661 INFO    MainThread:30500 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-20 16:27:32,661 INFO    MainThread:30500 [wandb_init.py:_log_setup():371] Logging user logs to C:\Users\rb01243\OneDrive - University of Surrey\Documents\GitHub\dental_disease\notebooks\wandb\run-20220120_162732-3uwmcb5s\logs\debug.log
2022-01-20 16:27:32,661 INFO    MainThread:30500 [wandb_init.py:_log_setup():372] Logging internal logs to C:\Users\rb01243\OneDrive - University of Surrey\Documents\GitHub\dental_disease\notebooks\wandb\run-20220120_162732-3uwmcb5s\logs\debug-internal.log
2022-01-20 16:27:32,661 INFO    MainThread:30500 [wandb_init.py:_jupyter_setup():321] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x0000026FA128D970>
2022-01-20 16:27:32,662 INFO    MainThread:30500 [wandb_init.py:init():404] calling init triggers
2022-01-20 16:27:32,662 INFO    MainThread:30500 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {'bcc_epoch': -1, 'qtfilter_epoch': -1, 'qt_thres_mode': '', 'qt_thres': 0.0, 'hybrid_entropy_thres': 0.0, 'hybrid_conf_thres': 0.0, 'weights': 'yolov5s.pt', 'cfg': '../src/models/yolov5s.yaml', 'data': '../data/iid-tvt.yaml', 'hyp': {'lr0': 0.01, 'lrf': 0.2, 'momentum': 0.937, 'weight_decay': 0.0005, 'warmup_epochs': 3.0, 'warmup_momentum': 0.8, 'warmup_bias_lr': 0.1, 'box': 0.05, 'obj': 1.0, 'cls': 0.5, 'cls_pw': 1.0, 'obj_pw': 1.0, 'iou_t': 0.2, 'anchor_t': 4.0, 'fl_gamma': 0, 'hsv_h': 0.015, 'hsv_s': 0.7, 'hsv_v': 0.4, 'degrees': 0.0, 'translate': 0.1, 'scale': 0.5, 'shear': 0.0, 'perspective': 0.0, 'flipud': 0.0, 'fliplr': 0.5, 'mosaic': 1.0, 'mixup': 0.0, 'copy_paste': 0.0}, 'epochs': 18, 'batch_size': 2, 'imgsz': 640, 'rect': False, 'resume': False, 'nosave': False, 'noval': False, 'noautoanchor': False, 'evolve': None, 'bucket': '', 'cache': None, 'image_weights': False, 'device': '', 'multi_scale': False, 'single_cls': False, 'adam': False, 'sync_bn': False, 'workers': 8, 'project': 'runs/train', 'entity': None, 'name': 'exp2099', 'exist_ok': False, 'quad': False, 'linear_lr': False, 'label_smoothing': 0.0, 'upload_dataset': False, 'bbox_interval': -1, 'save_period': -1, 'artifact_alias': 'latest', 'local_rank': -1, 'freeze': 0, 'patience': 1100, 'save_dir': 'runs\\train\\exp20998'}
2022-01-20 16:27:32,662 INFO    MainThread:30500 [wandb_init.py:init():460] starting backend
2022-01-20 16:27:32,662 INFO    MainThread:30500 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=spawn, using: spawn
2022-01-20 16:27:32,666 INFO    MainThread:30500 [backend.py:ensure_launched():216] starting backend process...
2022-01-20 16:27:32,731 INFO    MainThread:30500 [backend.py:ensure_launched():221] started backend process with pid: 14564
2022-01-20 16:27:32,732 INFO    MainThread:30500 [wandb_init.py:init():469] backend started and connected
2022-01-20 16:27:32,741 INFO    MainThread:30500 [wandb_run.py:_label_probe_notebook():911] probe notebook
2022-01-20 16:27:32,743 INFO    MainThread:30500 [wandb_run.py:_label_probe_notebook():921] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-01-20 16:27:32,743 INFO    MainThread:30500 [wandb_init.py:init():533] updated telemetry
2022-01-20 16:27:32,744 INFO    MainThread:30500 [wandb_init.py:init():563] communicating current version
2022-01-20 16:27:34,152 INFO    MainThread:30500 [wandb_init.py:init():568] got version response 
2022-01-20 16:27:34,152 INFO    MainThread:30500 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-20 16:27:34,562 INFO    MainThread:30500 [wandb_init.py:init():606] starting run threads in backend
2022-01-20 16:27:39,575 WARNING MainThread:30500 [wandb_init.py:init():952] interrupted
Traceback (most recent call last):
  File "C:\Users\rb01243\Anaconda3\lib\site-packages\wandb\sdk\wandb_init.py", line 931, in init
    run = wi.init()
  File "C:\Users\rb01243\Anaconda3\lib\site-packages\wandb\sdk\wandb_init.py", line 611, in init
    _ = backend.interface.communicate_run_start(run_obj)
  File "C:\Users\rb01243\Anaconda3\lib\site-packages\wandb\sdk\interface\interface.py", line 221, in communicate_run_start
    result = self._communicate_run_start(run_start)
  File "C:\Users\rb01243\Anaconda3\lib\site-packages\wandb\sdk\interface\interface_shared.py", line 423, in _communicate_run_start
    result = self._communicate(rec)
  File "C:\Users\rb01243\Anaconda3\lib\site-packages\wandb\sdk\interface\interface_shared.py", line 213, in _communicate
    return self._communicate_async(rec, local=local).get(timeout=timeout)
  File "C:\Users\rb01243\Anaconda3\lib\site-packages\wandb\sdk\interface\router.py", line 37, in get
    is_set = self._object_ready.wait(timeout)
  File "C:\Users\rb01243\Anaconda3\lib\threading.py", line 574, in wait
    signaled = self._cond.wait(timeout)
  File "C:\Users\rb01243\Anaconda3\lib\threading.py", line 316, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
2022-01-20 16:27:51,774 INFO    MainThread:30500 [wandb_settings.py:_apply_source_login():804] setting login settings: {}
2022-01-20 16:27:51,781 INFO    MainThread:30500 [wandb_setup.py:_flush():71] setting env: {}
2022-01-20 16:27:51,781 INFO    MainThread:30500 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-20 16:27:51,781 INFO    MainThread:30500 [wandb_init.py:_log_setup():371] Logging user logs to C:\Users\rb01243\OneDrive - University of Surrey\Documents\GitHub\dental_disease\notebooks\wandb\run-20220120_162751-izxqfemj\logs\debug.log
2022-01-20 16:27:51,781 INFO    MainThread:30500 [wandb_init.py:_log_setup():372] Logging internal logs to C:\Users\rb01243\OneDrive - University of Surrey\Documents\GitHub\dental_disease\notebooks\wandb\run-20220120_162751-izxqfemj\logs\debug-internal.log
2022-01-20 16:27:51,781 INFO    MainThread:30500 [wandb_init.py:init():404] calling init triggers
2022-01-20 16:27:51,782 INFO    MainThread:30500 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {'bcc_epoch': -1, 'qtfilter_epoch': -1, 'qt_thres_mode': '', 'qt_thres': 0.0, 'hybrid_entropy_thres': 0.0, 'hybrid_conf_thres': 0.0, 'weights': 'yolov5s.pt', 'cfg': '../src/models/yolov5s.yaml', 'data': '../data/iid-tvt.yaml', 'hyp': {'lr0': 0.01, 'lrf': 0.2, 'momentum': 0.937, 'weight_decay': 0.0005, 'warmup_epochs': 3.0, 'warmup_momentum': 0.8, 'warmup_bias_lr': 0.1, 'box': 0.05, 'obj': 1.0, 'cls': 0.5, 'cls_pw': 1.0, 'obj_pw': 1.0, 'iou_t': 0.2, 'anchor_t': 4.0, 'fl_gamma': 0, 'hsv_h': 0.015, 'hsv_s': 0.7, 'hsv_v': 0.4, 'degrees': 0.0, 'translate': 0.1, 'scale': 0.5, 'shear': 0.0, 'perspective': 0.0, 'flipud': 0.0, 'fliplr': 0.5, 'mosaic': 1.0, 'mixup': 0.0, 'copy_paste': 0.0}, 'epochs': 18, 'batch_size': 2, 'imgsz': 640, 'rect': False, 'resume': False, 'nosave': False, 'noval': False, 'noautoanchor': False, 'evolve': None, 'bucket': '', 'cache': None, 'image_weights': False, 'device': '', 'multi_scale': False, 'single_cls': False, 'adam': False, 'sync_bn': False, 'workers': 8, 'project': 'runs/train', 'entity': None, 'name': 'exp2099', 'exist_ok': False, 'quad': False, 'linear_lr': False, 'label_smoothing': 0.0, 'upload_dataset': False, 'bbox_interval': -1, 'save_period': -1, 'artifact_alias': 'latest', 'local_rank': -1, 'freeze': 0, 'patience': 1100, 'save_dir': 'runs\\train\\exp20999'}
2022-01-20 16:27:51,782 INFO    MainThread:30500 [wandb_init.py:init():460] starting backend
2022-01-20 16:27:51,782 INFO    MainThread:30500 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=spawn, using: spawn
2022-01-20 16:27:51,785 INFO    MainThread:30500 [backend.py:ensure_launched():216] starting backend process...
2022-01-20 16:27:51,849 INFO    MainThread:30500 [backend.py:ensure_launched():221] started backend process with pid: 23544
2022-01-20 16:27:51,850 INFO    MainThread:30500 [wandb_init.py:init():469] backend started and connected
2022-01-20 16:27:51,859 INFO    MainThread:30500 [wandb_run.py:_label_probe_notebook():911] probe notebook
2022-01-20 16:27:51,860 INFO    MainThread:30500 [wandb_run.py:_label_probe_notebook():921] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-01-20 16:27:51,860 INFO    MainThread:30500 [wandb_init.py:init():533] updated telemetry
2022-01-20 16:27:51,861 INFO    MainThread:30500 [wandb_init.py:init():563] communicating current version
2022-01-20 16:27:53,207 INFO    MainThread:30500 [wandb_init.py:init():568] got version response 
2022-01-20 16:27:53,207 INFO    MainThread:30500 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-20 16:27:53,581 INFO    MainThread:30500 [wandb_init.py:init():606] starting run threads in backend
2022-01-20 16:27:58,598 INFO    MainThread:30500 [wandb_run.py:_console_start():1810] atexit reg
2022-01-20 16:27:58,598 INFO    MainThread:30500 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.WRAP
2022-01-20 16:27:58,599 INFO    MainThread:30500 [wandb_run.py:_redirect():1721] Wrapping output streams.
2022-01-20 16:27:58,600 INFO    MainThread:30500 [wandb_run.py:_redirect():1745] Redirects installed.
2022-01-20 16:27:58,600 INFO    MainThread:30500 [wandb_init.py:init():633] run started, returning control to user process
2022-01-20 16:27:58,602 INFO    MainThread:30500 [wandb_run.py:_config_callback():956] config_cb None None {'data_dict': {'path': '../data/datasets/iid-tvt', 'train': '..\\data\\datasets\\iid-tvt\\images\\train', 'val': '..\\data\\datasets\\iid-tvt\\images\\val', 'test': '..\\data\\datasets\\iid-tvt\\images\\test', 'nc': 2, 'names': ['bone-loss', 'dental-caries']}}
